readme.md

```diff
@@ -456,6 +458,14 @@ test.cb(t => {
 });
 ```
 
+### Optional TAP output
+
+AVA can generate TAP output via `--tap` option for use with any 3rd-party reporters.
+
+```bash
+$ ava --tap | tap-spec
```

Comment:
Would be more fun to illustrate with the `nyan` reporter, right? You also need to show installing the reporter. Otherwise we'll end up with support issues on how it doesn't work...